examples/basic/main.tf

```diff
@@ -0,0 +1,19 @@
+data "aws_ecs_cluster" "example" {
+  cluster_name = "example"
+}
+
+data "aws_vpc" "initial" {}
+
+data "aws_vpc" "operations" {}
```

Comment:
i think for official examples, we should have an all-in-one example that always works, besides some parameters that the user can change. So we should rather make these data objects variables that the user can change.